internal/interconnections/create.go

```diff
@@ -58,14 +84,58 @@ func (c *Client) Create() *cobra.Command {
 	createInterconnectionsCmd.Flags().StringVarP(&metro, "metro", "m", "", "metro in the interconnection")
 	createInterconnectionsCmd.Flags().StringVarP(&redundancy, "redundancy", "r", "", "Website URL of the organization.")
 	createInterconnectionsCmd.Flags().StringVarP(&connType, "type", "t", "", "type of of interconnection.")
-	// createInterconnectionsCmd.Flags().StringVarP(&connType, "serviceTokentype", "T", "", "service token type for interconnection either fabric OR Metal builds")
-	createInterconnectionsCmd.Flags().StringSliceVarP(&vrfs, "vrfs", "v", []string{}, "Return only the specified vrfs.")
+	createInterconnectionsCmd.Flags().StringSliceVar(&vrfs, "vrfs", []string{}, "Array of strings VRF <uuid>.")
 	createInterconnectionsCmd.Flags().StringVarP(&projectID, "projectID", "p", "", "project ID")
 	createInterconnectionsCmd.Flags().StringVarP(&organizationID, "organizationID", "O", "", "Org ID")
+	createInterconnectionsCmd.Flags().Int32SliceVar(&vlans, "vlans", []int32{}, "Array of int vLANs")
```

Comment:
I think the argument name would be singular, and the user specifies multiple using multiple arguments:

`--vlan 1234 --vlan 9876`

I think Cobra also handles `--vlan 1234,9876` for us with `Int32SliceVar`.

I may be wrong about how we've previously named SliceVars. Follow the existing convention, whatever that is.

Reply:
Yes right. Making it singulare makes sense